.circleci/config.yml

```diff
@@ -1,4 +1,19 @@
 version: 2.1
+
+parameters:
+  GHA_Actor:
```

Comment:
not used?

Reply:
no, it's not is just the list of GHA params